docs/column_transforms/bin.md

```diff
@@ -2,6 +2,7 @@
 
 # bin
 
+(Snowflake-specific transform)
```

Comment:
Just curious why did we add this comment here? Is that because it's only possible in snowflake?

Would this cause potential problems for people using our library?

Reply:
this was just a test file. removing it